src/nodes/Networking/Inline.js

```diff
@@ -171,7 +171,7 @@ x3dom.registerNodeType(
             {
                 for ( var type in this.ContentType )
                 {
-                    if( contentType === this.ContentType[ type ] )
+                    if( type === contentType )
```

Comment:
if( contentType === this.ContentType[ type ] ) is the correct one here.
Input is "model/x3d+xml", "model/gltf+json", "model/gltf-binary" or "model/x3d+json"
The range of the documentation was wrong at that point